app/presenters/europe_at_work/index.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
```

Comment:
As this is identical to the migration presenter, for now just:
```ruby
# frozen_string_literal: true

module EuropeAtWork
  class Index < Migration::Index; end
end
```

Likewise for new.rb